harmoni_core/harmoni_recorder/requirements.txt

```diff
@@ -0,0 +1 @@
+pymongo
```

Comment:
I added this file, but we should update the relevant dockerfiles to add this dependency too.

Reply:
Agreed